main.js

```diff
@@ -15,3 +111,81 @@ const initApp = () => {
 document.addEventListener('DOMContentLoaded', () => {
   initApp();
 });
+
+// model
+
+const modelSection = document.querySelector('.model');
+const modelCloseButton = document.querySelector('.model-close');
+const overlay = document.getElementById('overlay');
+const projectButtons = document.querySelectorAll('.project-btn');
+
+function showPopup() {
+  modelSection.style.display = 'block';
+  overlay.style.display = 'block';
+}
+
+function hidePopup() {
+  modelSection.style.display = 'none';
+  overlay.style.display = 'none';
+}
+
+// document.querySelector(".btn-live").addEventListener("click", showPopup);
```

Comment:
- [optional] Please kindly remove this commented line of code, this will make your codebase clean. ℹ️

Reply:
done.

Reply:
done